tests/test_create.py

```diff
@@ -3,12 +3,14 @@
 """
 import os
 import nf_core.create
-import tempfile
 import unittest
 
+from .utils import with_temporary_folder
+
 
 class NfcoreCreateTest(unittest.TestCase):
-    def setUp(self):
+    @with_temporary_folder
```

Comment:
This is the decorator applied in `setUp`. Outside this function the directory is removed. It can work in this tests since we don't need the temporary folder anymore.